lib/Button/Button.tsx

```diff
@@ -0,0 +1,12 @@
+import React, { ReactNode } from 'react'
```

Comment:
Components should be under components directory unless it is a special pattern required from storybook.